NiL.JS/ExportTable.cs

```diff
@@ -23,12 +23,52 @@ public sealed class ExportTable : IEnumerable<KeyValuePair<string, JSValue>>
 
                 return result;
             }
-            internal set
+             set
             {
                 _items[key] = value;
             }
         }
+        /// <summary>
+        /// Add object constructor to export 
+        /// </summary>
+        /// <remarks>We dont recommend to use, because api can change and its can be broken in future. Use indexers</remarks>
+        /// <param name="type">Class</param>
+        /// <param name="name"></param>
+        public void DefineConstructor(Type type, string name)
+        {
+            var ctor = GlobalContext.CurrentGlobalContext.GetConstructor(type);
+            _items.Add(name, ctor);
+            ctor._attributes |= JSValueAttributesInternal.DoNotEnumerate;
+        }
 
+        /// <summary>
+        /// Add object to exports 
+        /// </summary>
+        /// <remarks>We dont recommend to use, because api can change and its can be broken in future. Use indexers</remarks>
+        /// <param name="name">Name of variable in export</param>
+        /// <param name="deletable"></param>
+        public JSValue DefineVariable(string name, bool deletable)
+        {
+            var defineVariable = GlobalContext.CurrentGlobalContext.DefineVariable(name, deletable);
```

Comment:
This line will work not how it's expected. Variable will be defined for all modules even without `import` command. It's not necessary to define variable in context in this case. This should be a simple value, stored in `_items`

Reply:
Thanks for say this 
I change very soon